py/kiwisolver/exceptions.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------
+# Copyright (c) 2023, Nucleic Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# --------------------------------------------------------------------------------------
+"""Kiwi exceptions."""
+
```

Comment:
Since this module is imported by the C extension (a rare thing), I would add a comment to the module docstring stating as such, along with a warning to not inadvertently trigger circular imports.

Reply:
I am fine adding a comment but I do not see how that could cause a circular import...